### ISSUE TITLE
Decrypt by authenticating with Fingerprint

### ISSUE THREAD

**Issue by metalmatze** [zeapo/Android-Password-Store#262]
Hey, I really like the app. I use it on a daily basis and it's perfect. 👌 
I thought about using the Fingerprint reader to decrypt the store. 1password does the same thing within their app. It might be really convenient. 
I'm not sure about the security implications though... 

Thanks!

**Comment by zeapo** [comment]
Hi,
Thank you for the kind comment.
The decryption using fingeprint, it's an OpenKeychain issue : https://github.com/open-keychain/open-keychain/issues/1642

They listed Keystore losing keys as an issue. We could suppose that we do not care about keystore losing our passphrase as the user should know the passphrase by heart, decrypt the passphrase and send it to Openkeychain...

**Comment by wiktor-k** [comment]
I think the fingerprint could be used when opening the app as a way to hide the list of used sites and available credentials. But on the other hand when using the accessibility service one can infer which sites have passwords.

Personally I wouldn't use it for decryption as this can be already protected using smartcard PIN and touch.

**Comment by raindev** [comment]
Looks like a duplicate of #156 (blocked on OpenKeychain).
